pkgs/tools/package-management/poetry/unwrapped.nix

```diff
@@ -54,6 +54,10 @@ buildPythonPackage rec {
     installShellFiles
   ];
 
+  postPatch = ''
+    substituteInPlace pyproject.toml --replace 'dulwich = "^0.21.2"' 'dulwich = "*"'
+  '';
```

Comment:
To quickly clarify, why not `pythonRelaxDepsHook`?

Reply:
Changed to use `pythonRelaxDepsHook`